ceph/cephfs/cephfs-provisioner.go

```diff
@@ -60,13 +61,16 @@ type cephFSProvisioner struct {
 	identity string
 	// Namespace secrets will be created in. If empty, secrets will be created in each PVC's namespace.
 	secretNamespace string
+	// set PVC quota
```

Comment:
prefer `enable quota`. `set PVC quota` could be confusing

Reply:
Please update comments here, then squash into one commit. Thanks for your patience! I tested it and it works very well.

cc @rootfs Any opinion?

Reply:
Everything is done now  ~~